plugins/woocommerce/includes/admin/helper/class-wc-helper-plugin.php

```diff
@@ -16,15 +16,25 @@
  * Contains the logic to manage the Woo Marketplace plugin.
  */
 class WC_Helper_Plugin {
-	const WOO_MARKETPLACE_PLUGIN_MAIN_FILE = 'woo-marketplace/woo-marketplace.php';
+	const WOO_UPDATE_MANAGER_PLUGIN_MAIN_FILE = 'woo-marketplace/woo-marketplace.php';
```

Comment:
In this file, there are different names for the same plugin:
- "Woo Marketplace plugin manager"
- WC_Helper_Updater
- WC_Helper_Plugin
- Woo Marketplace plugin
- WOO_UPDATE_MANAGER

Can these be brought to consistency with the official plugin name? 

Also, the filename `class-wc-helper-plugin.php` is confusing, as it manages the companion plugin (Woo Update Manager), while the WC Helper is a more general name of the whole system that manages connection, activations, etc.

Reply:
- "Woo Marketplace plugin manager" - replaced, after renaming the plugin main file
-  WC_Helper_Updater - wrong class here corrected.
- WC_Helper_Plugin - renamed to `WC_Woo_Update_Manager_Plugin`
- Woo Marketplace plugin - removed
- WOO_UPDATE_MANAGER - constants are left as it is.

Fixed with https://github.com/woocommerce/woocommerce/pull/44279/commits/2dd8ef6ea1afb35eeb41d839732044ad0cf8bcb5